docs/ff-concepts/alerts-notification/push-notifications.md

```diff
@@ -54,12 +50,18 @@ Here are the steps to send push notifications:
 
 To enable push notifications:
 
-1. Click on the **Settings and Integrations** -> App Settings -> **Push Notifications** and 
+1. Navigate to the **Settings and Integrations -> App Settings -> Push Notifications** and 
 **Enable Push Notifications**.
 2. Now, click on the **Deploy** button. This will create and deploy the *Cloud Functions* in your 
    Firebase project that are necessary for push notifications to work.
 
-![img.png](imgs/img.png)
+![img.png](imgs/enable-push-notification.avif)
+
+:::info
+By default, the **Automatically Prompt Users for Permission** option is enabled, meaning your app will automatically prompt users for permission to receive push notifications when FlutterFlow detects it's necessary. 
```

Comment:
"meaning your app will automatically prompt users requesting for permission to receive push notifications when the app is started. However, this may be disruptive to your user sign-in flow."